docs/proofs/output_alethe.rst

```diff
@@ -0,0 +1,8 @@
+Proof format: Alethe
+====================
+
+Using the flag :ref:`--proof-format-mode=alethe <lbl-option-proof-format-mode>`, cvc5 can output proofs in the `Alethe proof format <https://verit.loria.fr/documentation/alethe-spec.pdf>`_.
+
+Additonally, the following flags should be used to produce proofs with a minimal number of holes and no term sharing (which is not supported for Alethe yet): :ref:`--simplification=none <lbl-option-simplification>` :ref:`--dag-thresh=0 <lbl-option-dag-thresh>` :ref:`--proof-granularity=dsl-rewrite <lbl-option-proof-granularity>`.
```

Comment:
`dsl-rewrite` will not be supported in the relase.  It is only available on `proof-new`.

Reply:
So what should it say instead? (I don't actually feel responsible for the content :) )

Reply:
@Lachnitt

Reply:
I will just delete that part then. All THEORY_REWRITE will just be printed as all_simplifies which is fine.